samples/CMakeLists.txt

```diff
@@ -22,11 +22,18 @@ if (CONFIG_SPM)
 endif()
 
 if (CONFIG_SECURE_BOOT)
-  add_child_image(
-    NAME b0
-    SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/bootloader
-    )
-endif ()
+  if (CONFIG_SOC_NRF5340_CPUNET)
+    add_child_image(
+      NAME b0n
+      SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/nrf5340/netboot
+      )
```

Comment:
NCSDK-6234

Reply:
I don't quite get your point @tejlmand . `b0n` is the name of the image, regardless of the domain. The name is chosen at this point independently of domain, and I chose `b0n` simply because I found it suitable since its the `b0` of the `n`etwork core.
It is not similar to `b0` in functionality though, so re-using that name would not make sense either.

It would be great if you could elaborate :)

Reply:
i'm more thinking of:
```
if (CONFIG_SOC_NRF5340_CPUNET)
...
else()
...
endif()
```
for doing almost the same `b0n` vs `b0` and then a different `SOURCE_DIR`.
What will happen when a SoC `T` is made that either has another name than `b0<n>` or another `SOURCE_DIR`, and later SoC `Z` ?

A new `elseif` for each, where all they do is calling `add_child_image()` with little variation.